src/config.js

```diff
@@ -38,12 +38,20 @@ const getConfiguration = ({ constants, inputs } = {}) => {
     (inputs && inputs.thresholds) || process.env.THRESHOLDS || {};
 
   if (typeof thresholds === 'string') {
-    thresholds = JSON.parse(thresholds);
+    try {
+      thresholds = JSON.parse(thresholds);
+    } catch (e) {
+      throw new Error(`Invalid JSON for 'thresholds' input`);
```

Comment:
Should `e.message` be included in the thrown error message? This would allow the user to know which character failed.

Reply:
Fixed in https://github.com/netlify-labs/netlify-plugin-lighthouse/pull/80/commits/583090573efc79834f50cd8e2f609eabb10a0299